### PR TITLE
refactor: move workspace methods from App to MainWindow

### PR DESCRIPTION
**Pull request by rodolforg**
Workspace layout only matters/applies to `MainWindow`, no need to bloat `App` class with this stuff.

### CODE REVIEW

File: synfig-studio/src/gui/app.h [hunk 1]
```diff
@@ -353,13 +353,7 @@ class App : public Gtk::Application, private IconController
 	static const std::map<const char*, const char*>& get_default_accel_map();
 	static void load_file_window_size();
 	static void load_language_settings();
-	static void set_workspace_default();
-	static void set_workspace_compositing();
-	static void set_workspace_animating();
-	static void set_workspace_from_template(const std::string &tpl);
-	static void set_workspace_from_name(const std::string &name);
 	static void load_custom_workspaces();
```
Comment (ice0) [inline synfig-studio/src/gui/app.h:356]: Maybe move `load_custom_workspaces()`, `edit_custom_workspace_list()` and `WorkspaceHandler *studio::App::workspaces` too?

Comment (rodolforg) [reply synfig-studio/src/gui/app.h]: Done in a new commit

**Comment by rodolforg** [comment]
(rebased)

**Review by ice0 — APPROVED** [review]

**Comment by ice0** [comment]
Merged. Thank you!

**Pull request merged**